tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -1519,9 +1519,9 @@ public function test_checkout_saves_sepa_generated_payment_method_to_order() {
 	 * @dataProvider provider_handle_saving_payment_method_respects_per_method_toggle
 	 */
 	public function test_handle_saving_payment_method_respects_per_method_toggle( string $sepa_tokens_for_ideal, bool $expected_save ) {
-		$stripe_settings                          = WC_Stripe_Helper::get_stripe_settings();
+		$stripe_settings                          = WC_Stripe::read_settings_option();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe' in method 'test_handle_saving_payment_method_respects_per_method_toggle'. (undefined)

(StaticAccess)